Meter.c

```diff
@@ -50,8 +53,10 @@ Meter* Meter_new(const struct ProcessList_* pl, unsigned int param, const MeterC
    return this;
 }
 
+static const char* const Meter_prefixes = "KMGTPEZY";
```

Comment:
FYI: https://phys.org/news/2022-11-earth-ronnagrams-metric-prefixes-voted.html

Reply:
Will address this issue in #1321.